src/lib/core/base/base2.ts

```diff
@@ -78,10 +86,20 @@ export abstract class BaseDirective2 implements OnChanges, OnDestroy {
       }
     }
 
+    this.mru = {...genStyles};
     this.applyStyleToElement(genStyles);
     builder.sideEffect(input, genStyles, parent);
   }
 
+  /** Remove generated styles from an element using predefined style builder */
+  protected clearStyles() {
```

Comment:
Under what conditions should `clearStyles()` be called?

Reply:
When there isn't fallback value for a given element and key, it indicates that there is no actual value for the activated breakpoints, meaning no styles should be applied. To accomplish this, we clear out the most recently used styles by calling this method.